### ISSUE TITLE
PTT by RTS or CAT?

### ISSUE THREAD

**Issue by yeti7** [raytheonbbn/hammer#18]
Hi, I'm Denis (K0TX) - a ham behind [digirig project](https://github.com/softcomplex/digirig).
The interface's sound card works with Hammer Plugin, but there is also a serial COM port that allows PTT by RTS signal or CAT with the capable radios. My question is if it would be possible to modify the plugin to allow these additional more reliable PTT methods?

**Comment by robertcedwards** [comment]
Yes! This is what I was searching for...has there been any headway with Digirig support?
